requirements.txt

```diff
@@ -1,10 +1,3 @@
 boto
 boto3
 botocore
-
-[dev]
-check-manifest
-
-[test]
```

Comment:
Why did you remove tests?

Reply:
This syntax wasn't valid for me.  Per the instructions in README and general practice:

```
 $  pip install -r requirements.txt 
Invalid requirement: '[dev]'
Traceback (most recent call last):
  File "/Users/jhart/rapid7/awsretry/venv/lib/python2.7/site-packages/pip/req/req_install.py", line 82, in __init__
    req = Requirement(req)
  File "/Users/jhart/rapid7/awsretry/venv/lib/python2.7/site-packages/pip/_vendor/packaging/requirements.py", line 96, in __init__
    requirement_string[e.loc:e.loc + 8]))
InvalidRequirement: Invalid requirement, parse error at "u'[dev]'"
```

Reply:
I believe I've fixed this.  Thanks for looking at this PR.  Really hoping to get this all fixed so we can more readily use this in some of our projects.